openlibrary/templates/type/author/view.html

```diff
@@ -194,7 +198,7 @@ <h6 class="collapse black uppercase">$label</h6>
                 $:render_template("lists/widget", page, include_rating=False)
             </div>
 
-        <div class="section">
+        <div class="section" itemscope itemtype="https://schema.org/Thing">
```

Comment:
shouldn't this be `itemtype="https://schema.org/Person"` ?

Reply:
Oh ok, will change it. But as mentioned in the comment [here](https://github.com/internetarchive/openlibrary/issues/3231#issuecomment-602078002) that all authors are not people, so I went for `Thing`. But will change it :).